static/styles/popovers.css

```diff
@@ -703,7 +703,19 @@ ul {
     .status_emoji {
         height: 18px;
         width: 18px;
-        top: 2px;
+        top: 5px;
+        left: -4px;
+        margin-right: 1px;
+    }
+
+    #status_message {
+        display: flex;
+        align-items: baseline;
+        hyphens: auto;
```

Comment:
There is no easy way to add hyphens here. The wrapping works well, though.
<img width="195" alt="image" src="https://user-images.githubusercontent.com/25124304/193762494-80184155-cff9-4366-b509-34a5a8cf456e.png">

Reply:
This is certainly much better than the current situation. :)